e2e/steps_file.js

```diff
@@ -153,6 +153,15 @@ module.exports = function () {
       this.dontSeeElement(caseListPage.locateCase(normalizeCaseId(caseId)));
     },
 
+    async seeEndStateForEvent(eventName, state) {
+      try {
+        await this.waitForSelector(`//tr[@class="EventLogTable-Selected" and td[contains(., "${eventName}")]]`);
+      } catch (notFound) {
+        this.click(`//table[@class="EventLogTable"]//tr[td[contains(., "${eventName}")]][1]`);
```

Comment:
cannot we always click on a row and then check? it looks dodgy a bit with catch

Reply:
probably, will give it a go. Myself and joe also thought it looked dodgy.

Reply:
do you think it would be worth adding an optional index parameter? Speaking of which the index should probably be `[0]` not `[1]`

Reply:
turns out I am wrong on that last part, xpath has a 1 based index